app/models/user.rb

```diff
@@ -212,9 +212,9 @@ def generate_password
     self.password = self.password_confirmation = last_name
   end
 
-  def generate_student
+  def generate_student(classroom_id)
```

Comment:
need to pass a classroom_id argument when generating student, as they can have multiple classrooms